docs/project/changelog.md

```diff
@@ -63,6 +63,7 @@ myst:
 - Upgraded `scipy` to 1.13.0 {pr}`4719`
 - Added `casadi` 3.6.5 {pr}`4936`
 - Added `rasterio` 1.13.10, `affine` 2.4.0 {pr}`4983`
+- Upgraded `scikit-image` to 0.24.0 {pr}`5003`
```

Comment:
Could you please move this up so that it is below one of the "Upgraded" entries and above the "Added" ones?

Reply:
Sure! Done 49b748cb8e7ee7a264f11.